ui/hunter/beast_mastery/sim.ts

```diff
@@ -118,7 +118,7 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecBeastMasteryHunter, {
 	},
 
 	// IconInputs to include in the 'Player' section on the settings tab.
-	playerIconInputs: [HunterInputs.PetTypeInput(), HunterInputs.WeaponAmmo(), HunterInputs.UseHuntersMark()],
+	playerIconInputs: [HunterInputs.PetTypeInput(), HunterInputs.UseHuntersMark()],
```

Comment:
Do we want to have Hunters Mark be a player input? Isnt it better if its an actual spell that can be cast with a prepull action?

Reply:
Since it's a hunter specific debuff, would I have to define it in debuffs.go still, or can I just apply the debuff sort of like CoE using the psuedostat `BonusRangedAttackPowerTaken` at the OnSpellHitDealt in a Hunters mark spell?

Reply:
Hunters mark should still be implemented in debuffs.go and exposed in the debuffs section because other sims want to be able to activate it (unless the spell is changed in cata to have no effect for other classes).

What i meant here is that instead of controlling if the aura is active or not with a player option for the hunter sim specifically you should define a hunters mark spell that just applies the debuff and users can do it from APL

Reply:
See my implementation